pycript/Reqcheck.py

```diff
@@ -110,20 +110,9 @@ def decrypt_and_update_parameters(extender, currentreq, decryptionpath, selected
                 output = extender.helpers.stringToBytes(dumps(json_object,separators=(',', ':')))
                 currentreq = extender.helpers.buildHttpMessage(header, output)
                 break
-
-        # if BOTH is selected first update GET param values, then form body
+        
         else:
-            if param.getType() == IParameter.PARAM_URL:
-                    decrypted_param =  Parameterdecrypt(selectedlang, decryptionpath, param.getValue())
-                    currentreq = extender.helpers.updateParameter(currentreq, extender.helpers.buildParameter(param.getName(), decrypted_param, param.getType()))
-
-            elif param.getType() == IParameter.PARAM_BODY:
-                decrypted_param =  Parameterdecrypt(selectedlang, decryptionpath, param.getValue())
-                currentreq = extender.helpers.updateParameter(currentreq, extender.helpers.buildParameter(param.getName(), decrypted_param, param.getType()))
-
-    # get the updated parameters from the update request
-    parameters = extender.helpers.analyzeRequest(currentreq).getParameters()
-    header = extender.helpers.analyzeRequest(currentreq).getHeaders()
+            print("Something went wrong, Please check the request and logger")
```

Comment:
the logic is incorrect. 
1. You don't need to remove "Both" from the current loop for GET and Body
2. Print is not required value is set from the radio button and will always have 3 values else condition will triggered as error for "Both" as selected method.